pt-client-2/pt-client-2/pt_firmware_download_api_internal.h

```diff
@@ -26,53 +26,51 @@
 #include "pt-client-2/pt_common_api_internal.h"
 #include "arm_uc_public.h"
 
+#define COMPONENT 14
+#define COMPONENT_INSTANCE 0
+#define COMPONENT_NAME 0
+#define COMPONENT_VERSION 2
+
+#define COMPONENT_NAME_STR "MAIN"
+#define VERSION_NAME_STR "0.0.0"
 #define MANIFEST_VENDOR_STR "SUBDEVICE-VENDOR"
 #define MANIFEST_VENDOR_STR_SIZE strlen(MANIFEST_VENDOR_STR)
-#define MANIFEST_CLASS_STR "SUBDEVICE-CLASS"
+#define MANIFEST_CLASS_STR "SUBDEVICE--CLASS"
```

Comment:
@prakharbhatt007 Let's make this configurable through CMake flags. This will allow users to modify it at the build time. Also, accordingly update the README of the C example.

Reply:
@ygoyal18 I was thinking, that this should be configured by mbed-edge-example because the vendor and class information should be given by the example only. This flag should be configured by example only.